docs/release/README.md

```diff
@@ -0,0 +1,59 @@
+# Releasing the Training Operator
+
+## Prerequisite
+
+1. Permissions
+   - You need write permissions on the repository to create a release tag/branch.
+1. Prepare your Github Token
+
+1. Install Github python dependencies to generate changlog
+   ```
+   pip install PyGithub==1.55
+   ```
+
+### Release Process
+
+1. Make sure the last commit you want to release past `kubeflow-training-operator-postsubmit` testing.
```

Comment:
Is this still needed? IIUC, the prow doesn't exist, right?

Reply:
Oh, good point. I will fix this Readme in followup PR.